uint32 count
geometry_msgs/Point[] points_2d
geometry_msgs/Point[] points_3d
